README.md

```diff
@@ -206,6 +206,17 @@ The shapefile used to create the Michigan ArcGIS server can be found here: http:
 7. On the bottom of the page click on the `Query` link
 8. You will be prompted to a page where you can visually input your query parameters, after you are done inputing them you can click on the `Query (GET)` button on the bottom of the page and you will navigate to your endpoint
 
+## Logging and Webhooks
+We use Google Cloud Logging and Mailgun Webhooks to log and keep track of sent emails. The instructions to enable and set up Google Cloud Logging can be found [here](https://cloud.google.com/logging/docs/setup/nodejs#before_you_begin), and an article for setting up Mailgun webhooks is available [here](https://www.mailgun.com/blog/a-guide-to-using-mailguns-webhooks/).
+
+List of Mailgun webhooks used by VoteByMail.io:
+
+  type                | value
+  --------------------|-------------------------------------
+  **complained**      | REACT_APP_SERVER/mailgun_logging_webhook
```

Comment:
@ludanin These are what you set up through the Mailgun UI on your personal test account, right? I think I'd like to actually just log all available events -- not that it matters for anything but this documentation, but wanted to make sure I'm not missing something in the code.

Reply:
I thought we were only logging errors, besides documentation there's a small change at `mg.ts`, thanks for letting me know